doc/classes/RayCast2D.xml

```diff
@@ -56,6 +56,21 @@
 			<return type="int" />
 			<description>
 				Returns the shape ID of the first object that the ray intersects, or [code]0[/code] if no object is intersecting the ray (i.e. [method is_colliding] returns [code]false[/code]).
+				To get the intersected shape node, for a [CollisionObject2D] target, use:
+				[codeblocks]
+				[gdscript]
+				var target = get_collider() # A CollisionObject2D.
```

Comment:
Instead of a comment that can be misleading (it could be a TileMap-related object) can you just add `if target is CollisionObject2D:` in the next line?